### PR TITLE
[common] New Search Engine: TorrentDownloads.me

### PR DESCRIPTION
**Pull request by alejandroarturom**

### CODE REVIEW

File: common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        return new TorrentDownloadsTempSearchResult(getDomainName(), itemId);
+    }
+
+    @Override
+    protected int htmlPrefixOffset(String html) {
+        int offset = html.indexOf("Torrent Search Results<span>");
+        return offset > 0 ? offset : 0;
+    }
+/*
+    @Override
```
Comment (gubatron) [inline common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java]: and the suffix?

File: common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+package com.frostwire.search.torrentdownloads;
+
+import com.frostwire.regex.Pattern;
+import com.frostwire.search.CrawlableSearchResult;
+import com.frostwire.search.ScrapedTorrentFileSearchResult;
+import com.frostwire.search.SearchMatcher;
+import com.frostwire.search.SearchResult;
+import com.frostwire.search.torrent.TorrentCrawlableSearchResult;
+import com.frostwire.search.torrent.TorrentRegexSearchPerformer;
+import org.apache.commons.io.FileUtils;
```
Comment (gubatron) [inline common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java]: clean the imports using "Organize Imports"

File: common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+import java.io.File;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * @author gubatron
+ * @author aldenml
+ */
```
Comment (gubatron) [inline common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java]: please add yourself as an author on all of the files you added

File: common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
+
+
+/**
+ * Created by alejandroarturom on 23-05-16.
+ */
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Copyright (c) 2011-2015, FrostWire(R). All rights reserved.
```
Comment (gubatron) [inline common/src/main/java/com/frostwire/search/torrentdownloads/TorrentDownloadsSearchPerformer.java]: replace `2015` for `2016`

**Comment by alejandroarturom** [comment]
Done @gubatron

**Comment by gubatron** [comment]
Thank you.

We should keep this patch as an example of how to add a new Search Engine to FrostWire and talk about it on the developer blog at wordpress

**Pull request merged**